changelog/30683.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
For the enterprise changelog entry, I think we've decided this is a feature actually